modules/ROOT/pages/customize-nic.adoc

```diff
@@ -1,7 +1,7 @@
 = How to Customize a NIC Name
 
 == Using a systemd Link File
-You can create a systemd xref:https://www.freedesktop.org/software/systemd/man/systemd.link.html[link file] with Ignition configs.
+You can create a systemd link:https://www.freedesktop.org/software/systemd/man/systemd.link.html[link file] with Ignition configs.
```

Comment:
Looking at other pages in the repository, I think you can write this hyperlink as `https://www.freedesktop.org/software/systemd/man/systemd.link.html[link file]` and it will work.
That is, directly drop the `xref:` prefix in the original file.

Reply:
> That is, directly drop the `xref:` prefix in the original file.

For example:

```suggestion
You can create a systemd https://www.freedesktop.org/software/systemd/man/systemd.link.html[link file] with Ignition configs.
```

Reply:
Yeah, I know, I was just to lazy, I mean it also works. Fixed now. :upside_down_face: